firmware/boron-ins-fsm/test/base.h

```diff
@@ -8,17 +8,26 @@
 #include "mocks/eeprom.h"
 #include "mocks/Particle.h"
 #include "mocks/System.h"
+#include "mocks/os_queue_t.h"
+#include "mocks/mock_ble.h"
+
+// Particle.h library include files
+// Copied from local Particle toolchain files
```

Comment:
This is a good point. I imagine they could potentially change when upgrading the device OS, and it would probably be safer to recopy them. I'll add some info to the documentation